crates/astria-bridge-withdrawer/src/withdrawer/ethereum/convert.rs

```diff
@@ -132,7 +135,8 @@ fn event_to_ics20_withdrawal(
     let denom = rollup_asset_denom.clone();
 
     let channel = denom
-        .channel()
+        .as_trace_prefixed()
+        .and_then(TracePrefixed::last_channel)
         .ok_or_eyre("denom must have a channel to be withdrawn via IBC")?;
```

Comment:
Noticing this: the service does not actually enforce any form of the provided ics20 denom otherwise, but this error seems to suggest that `ibc/<hash>` denoms are not valid?

If that's the case the watcher builder can just parse the config input as `TracePrefixed` to test this at startup.